src/pint/utils.py

```diff
@@ -1580,21 +1599,206 @@ def companion_mass(pb, x, inc=60.0 * u.deg, mpsr=1.4 * u.solMass):
     return x1.to(u.Msun)
 
 
-def ELL1_check(A1, E, TRES, NTOA, outstring=True):
+@u.quantity_input
+def PBDOT(mp: u.Msun, mc: u.Msun, pb: u.d, e: u.dimensionless_unscaled):
+    """Post-Keplerian orbital decay PBDOT.
+
+    Parameters
+    ----------
+    mp : astropy.units.Quantity
+        pulsar mass
+    mc : astropy.units.Quantity
+        companion mass
+    pb : astropy.units.Quantity
+        Binary orbital period
+    e : Quantity or float
+        orbital eccentricity
+
+    Returns
+    -------
+    pbdot : astropy.units.Quantity
+        (dimensionless)
+
+    Raises
+    ------
+    astropy.units.UnitsError
+        If the input data are not appropriate quantities
+    TypeError
+        If the input data are not quantities
+
+    Notes
+    -----
+    Calculates :math:`(-192\pi/5)T_{\odot}^{5/3} (P_b/2\pi)^{-5/3} f(e)m_p m_c (m_p+m_c)^{-1/3}`,
+    with :math:`f(e)=(1+(73/24)e^2+(37/96)e^4)(1-e^2)^{-7/2}` and :math:`T_\odot = GM_\odot c^{-3}`.
+
+    See [1]_
+
+    .. [1] Lorimer & Kramer, 2008, "The Handbook of Pulsar Astronomy", Eqn. 8.52
+
+    """
+    f = (1 + (73.0 / 24) * e ** 2 + (37.0 / 96) * e ** 4) / (1 - e ** 2) ** (7.0 / 2)
+    value = (
+        (const.G / const.c ** 3) ** (5.0 / 3)
+        * (pb / (2 * np.pi)) ** (-5.0 / 3)
+        * (-192 * np.pi / 5)
+        * f
+        * (mp * mc)
+        / (mp + mc) ** (1.0 / 3)
+    )
+    return value.to(u.s / u.s)
+
+
+@u.quantity_input
+def GAMMA(mp: u.Msun, mc: u.Msun, pb: u.d, e: u.dimensionless_unscaled):
+    """Post-Keplerian time dilation and gravitational redshift GAMMA
```

Comment:
I think this one could really use some explanation - in particular that it has no relationship to the PPN parameter gamma, and also how it enters into the PPK equations of motion?

Reply:
also done.  note that this (and the others) do include citations to the literature